packages/core/src/interpreter.ts

```diff
@@ -313,7 +313,7 @@ export class Actor<TLogic extends AnyActorLogic>
     this.system._sendInspectionEvent({
       type: '@xstate.event',
       sourceRef: this._parent,
-      targetRef: this,
+      actorRef: this,
```

Comment:
I see the argument of having a "common" `actorRef` but when looking at this particular event in isolation... I think that the `sourceRef` and `targetRef` were more readable. I see `actorRef` here and I'm like - ye, sure, but which one?

Reply:
Can sorta be a process-of-elimination though: if it's not the source, it must be the target